page/docs/guide/advancing-time.md

```diff
@@ -7,7 +7,7 @@ Block timestamp can be manipulated by setting the exact time or setting the time
 
 ## Set time
 
-Sets the exact time of the next generated block. All subsequent blocks will keep the set offset.
+Sets the exact time and generate a new block. All subsequent blocks will keep the set offset.
```

Comment:
Should we include info about request failing if there are pending blocks?

Reply:
true